pages/common/git-ignore.md

```diff
@@ -1,12 +1,20 @@
 # git ignore
 
-> Generate .gitignore files from predefined templates.
-> More information: <https://docs.gitignore.io/install/command-line>.
+> Show/update `.gitignore` files.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-ignore>.
 
-- List available templates:
+- Show global and local `.gitignore` files' content:
```

Comment:
```suggestion
- Show the content of  all global and local `.gitignore` files:
```
I think this is a bit more readable. Also, does `global` mean, that it changes it for all git repos?

Reply:
Not to sure if this is correct however, is `global` indicating all ALL branches have it ignored